### ISSUE TITLE
accessibility: update role for SVG icons

### ISSUE THREAD

**Issue by Trott** [tomusborne/generatepress#119]
The SVG icons have `aria-hidden="true" role="img"`. 

If I understand correctly, the `aria-hidden="true"` is, basically "Hey, assistive technology, ignore this. It doesn't matter." But the `role="img"` might suggest otherwise. If I'm not mistaken, it should be `role="presentation"` or something similar to that. 

If it adds any weight, the SiteImprove tool is flagging the SVG icons as an accessibility issue.

**Comment by Trott** [comment]
A colleague reproduced this at https://generatepress.com/forums/topic/role-for-svg-icons-causes-accessibility-fail/ and I see from the information in the reply there that this is an issue with SiteImprove and/or FontAwesome and there's unlikely much GeneratePress can do. So I'll go ahead and close this.

/cc @mizejewski
